platform-operator/platformcontroller.go

```diff
@@ -847,6 +893,19 @@ func deleteCRDInstances(kind, group, version, plural, namespace string) []byte {
 }
 
 
+func updateCRDInstances(kind, group, version, plural, namespace, chartURL, chartName string) []byte {
+	fmt.Printf("Inside updateCRDInstances...\n")
+	args := fmt.Sprintf("kind=%s&group=%s&version=%s&plural=%s&namespace=%s&chartURL=%s&chartName=%s", kind, group, version, plural, namespace, chartURL, chartName)
+	//serviceHost, servicePort := getServiceEndpoint("kubeplus")
```

Comment:
You can delete these commented lines in this method. 

They are coming from an earlier design. Other methods also have these commented lines, which in a separate PR we can remove. For new code, we can start clean from the outset.